### PR TITLE
update retl extract schedule

### PR DESCRIPTION
**Pull request by terence1988**
### Proposed changes
We have updated the product but docs is outdated

### Merge timing
ASAP once approved

### CODE REVIEW

File: src/connections/reverse-etl/index.md [hunk 1]
```diff
@@ -96,32 +96,33 @@ To create a mapping:
...
-      * You map the fields that come from your source, to fields that the destination expects to find. Fields on the destination side depend on the type of action selected.
-      * If you're setting up a destination action, depending on the destination, some mapping fields may require data to be in the form of an object or array. See the [supported objects and arrays for mapping](#supported-object-and-arrays).
-9. *(Optional)* Send a test record to verify the mappings correctly send to your destination. 
-9. Click **Save Mapping**.
-10. Select the destination you’d like to enable the mapping for under **Connections > Destinations > Reverse ETL**. 
-11. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.
+    * For a **Day and time** schedule type, you can choose the day(s) you’d like the schedule to sync as well as the time.
+        * You can only choose to start the **Extraction** at the top of the hour.
+        * Scheduling multiple **Extractions** to start at the same time inside the same data warehouse can cause extraction errors
```
Comment (stayseesong) [inline src/connections/reverse-etl/index.md:116]:
```suggestion
        * Scheduling multiple extractions to start at the same time inside the same data warehouse causes extraction errors.
```

File: src/connections/reverse-etl/index.md [hunk 2]
```diff
@@ -96,32 +96,33 @@ To create a mapping:
...
-8. Define how to map the record columns from your model to your destination in the **Select Mappings** section.
-      * You map the fields that come from your source, to fields that the destination expects to find. Fields on the destination side depend on the type of action selected.
-      * If you're setting up a destination action, depending on the destination, some mapping fields may require data to be in the form of an object or array. See the [supported objects and arrays for mapping](#supported-object-and-arrays).
-9. *(Optional)* Send a test record to verify the mappings correctly send to your destination. 
-9. Click **Save Mapping**.
-10. Select the destination you’d like to enable the mapping for under **Connections > Destinations > Reverse ETL**. 
-11. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.
+    * For a **Day and time** schedule type, you can choose the day(s) you’d like the schedule to sync as well as the time.
+        * You can only choose to start the **Extraction** at the top of the hour.
```
Comment (stayseesong) [inline src/connections/reverse-etl/index.md:115]:
```suggestion
        * You can only choose to start the extraction at the top of the hour.
```

File: src/connections/reverse-etl/index.md [hunk 3]
```diff
@@ -96,32 +96,33 @@ To create a mapping:
...
+        * Scheduling multiple **Extractions** to start at the same time inside the same data warehouse can cause extraction errors
+10. Define how to map the record columns from your model to your destination in the **Select Mappings** section.
+    * You map the fields that come from your source, to fields that the destination expects to find. Fields on the destination side depend on the type of action selected.
+    * If you're setting up a destination action, depending on the destination, some mapping fields may require data to be in the form of an object or array. See the [supported objects and arrays for mapping](#supported-object-and-arrays).
+11. Click **Create Mapping**.
+12. Select the destination you’d like to enable on the **My Destinations** page under **Reverse ETL > Destinations**.
```
Comment (stayseesong) [inline src/connections/reverse-etl/index.md:121]:
```suggestion
13. Select the destination you’d like to enable on the **My Destinations** page under **Connections > Destinations> Reverse ETL**.
```

File: src/connections/reverse-etl/index.md [hunk 4]
```diff
@@ -96,32 +96,33 @@ To create a mapping:
...
-9. Click **Save Mapping**.
-10. Select the destination you’d like to enable the mapping for under **Connections > Destinations > Reverse ETL**. 
-11. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.
+    * For a **Day and time** schedule type, you can choose the day(s) you’d like the schedule to sync as well as the time.
+        * You can only choose to start the **Extraction** at the top of the hour.
+        * Scheduling multiple **Extractions** to start at the same time inside the same data warehouse can cause extraction errors
+10. Define how to map the record columns from your model to your destination in the **Select Mappings** section.
+    * You map the fields that come from your source, to fields that the destination expects to find. Fields on the destination side depend on the type of action selected.
+    * If you're setting up a destination action, depending on the destination, some mapping fields may require data to be in the form of an object or array. See the [supported objects and arrays for mapping](#supported-object-and-arrays).
+11. Click **Create Mapping**.
```
Comment (stayseesong) [inline src/connections/reverse-etl/index.md:120]:
```suggestion
11. *(Optional)* Send a test record to verify the mappings correctly send to your destination.
12. Click **Create Mapping**.
```

File: src/connections/reverse-etl/index.md [hunk 5]
```diff
@@ -96,32 +96,33 @@ To create a mapping:
...
+12. Select the destination you’d like to enable on the **My Destinations** page under **Reverse ETL > Destinations**.
+13. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.
```
Comment (stayseesong) [inline src/connections/reverse-etl/index.md:122]:
```suggestion
14. Turn the toggle on for the **Mapping Status**. Events that match the trigger condition in the mapping will be sent to the destination.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by terence1988** [comment]
Thanks, I have updated the content

**Review by stayseesong — APPROVED** [review]

**Pull request merged**